645b215a29f36ff7921f0180d9477a8a15df88b4
apollo-biz/src/test/java/com/ctrip/framework/apollo/biz/service/ClusterServiceTest.java
--- 
+++ 
@@ -1,53 +1,53 @@
 /*
- * Copyright 2022 Apollo Authors
+ * Copyright 2023 Apollo Authors
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
  * http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  *
  */
 package com.ctrip.framework.apollo.biz.service;
 
 import com.ctrip.framework.apollo.biz.AbstractIntegrationTest;
 import com.ctrip.framework.apollo.common.entity.App;
 import com.ctrip.framework.apollo.common.exception.ServiceException;
 
 import org.junit.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 
 import java.util.Date;
 
 public class ClusterServiceTest extends AbstractIntegrationTest {
 
   @Autowired
   private AdminService adminService;
 
   @Autowired
   private ClusterService clusterService;
 
   @Test(expected = ServiceException.class)
   public void testCreateDuplicateCluster() {
     String appId = "someAppId";
     App app = new App();
     app.setAppId(appId);
     app.setName("someAppName");
     String owner = "someOwnerName";
     app.setOwnerName(owner);
     app.setOwnerEmail("someOwnerName@ctrip.com");
     app.setDataChangeCreatedBy(owner);
     app.setDataChangeLastModifiedBy(owner);
     app.setDataChangeCreatedTime(new Date());
 
     adminService.createNewApp(app);
 
     clusterService.createDefaultCluster(appId, owner);
   }
 }